freediscovery/server/resources.py

```diff
@@ -846,3 +848,31 @@ def post(self, **args):
         res = model.fe.db.render_dict(scores_pd)
 
         return {'data': res}
+
+
+# ============================================================================ #
+#                            Custom Stop Words
+# ============================================================================ #
+
+
+class CustomStopWordsApi(Resource):
+    @doc(description="Perform a mechanism for adding / managing custom stop words")
+    @use_args({"name" : wfields.Str(required=True),
+               "stop_words" : wfields.List(wfields.Str(), required=True)})
+    @marshal_with(CustomStopWordsSchema())
+    def post(self, **args):
+        self.name = args['name']
```

Comment:
This method is stateless, meaning it only depends on the input parameters, so we don't need to save anything to `self`. It could be just `name  = args['name']`, etc..